src/sass/partials/components/_breadcrumb.scss

```diff
@@ -0,0 +1,43 @@
+.breadcrumb {
```

Comment:
One last nitpick now that I'm reading this over one last time; the component HTML is `breadcrumbs` and the styling is `breadcrumb`. These should be consistent